src/constants.py

```diff
@@ -5,14 +5,23 @@
 
 CHECK_READY_NAME = "synapse-ready"
 CHECK_ALIVE_NAME = "synapse-alive"
+CHECK_NGINX_READY_NAME = "synapse-nginx-ready"
```

Comment:
I think the constants defined here that are only used in pebble.py would be better defined there

Reply:
They are used in the workload too :)

Reply:
Ups, my bad